### DISCUSSION TITLE
edit question

### DISCUSSION THREAD

**Discussion by handsome-xiaowang** [excalidraw/excalidraw#8667] [General]
Our requirement is that there is no need to change the size of elements during editing; Does this plugin support it

**Comment by dwelle** [comment]
Hi, do you mean element locking? You can do that by right-clicking it -> lock element.

![Image](https://github.com/user-attachments/assets/e5d4189a-9402-480a-a474-23324e06cf8a)

**Reply by handsome-xiaowang** [reply]
Is there any other way besides locking to achieve the goal of not changing the size of elements

**Reply by handsome-xiaowang** [reply]
He can still drag elements

**Reply by dwelle** [reply]
Not currently, but it's something we may support in the future.
